README.md

```diff
@@ -90,6 +90,7 @@ Grafana ships with built-in support for Loki in the [latest nightly builds](http
 3. Click the `+ Add data source` button in the top header.
 4. Choose Loki from the list.
 5. The http URL field should be the address of your Loki server e.g. `http://localhost:3100` and `http://loki:3100` when running with docker and docker-compose.
+6. Click the `Explore` not `Dashboard`, enter `__filename__="/var/log/docker.log"` then you can get docker log in UI.
```

Comment:
Let's rephrase this to stay independent on what labels loki provides. I suggest:

`6. To see the logs, click "Explore" on the sidebar, select the Loki datasource, and then choose a log stream using the "Log labels" button.`

Reply:
done as you suggest.